python/python/glide/async_commands/core.py

```diff
@@ -2877,6 +2878,57 @@ async def zdiffstore(self, destination: str, keys: List[str]) -> int:
             ),
         )
 
+    async def bzmpop(
+        self,
+        keys: List[str],
+        modifier: ScoreFilter,
+        timeout: float,
+        count: Optional[int] = None,
+    ) -> Optional[List[Union[str, Dict[str, float]]]]:
+        """
+        Pops a member-score pair from the first non-empty sorted set, with the given keys being checked in the order
+        that they are given. Blocks the connection when there are no members to pop from any of the given sorted sets.
+        The optional `count` argument can be used to specify the number of elements to pop, and is set to 1 by default.
+        The number of popped elements is the minimum from the sorted set's cardinality and `count`.
+
+        When in cluster mode, all keys must map to the same hash slot.
+
+        `BZMPOP` is the blocking variant of `ZMPOP`.
+
+        `BZMPOP` is a client blocking command, see https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands for more details and best practices.
```

Comment:
so about https://github.com/aws/glide-for-redis/pull/1399#discussion_r1598386730
I think because we have some commands with this type of documentation (maybe unlink if i remember right)
I think it wont hurt if we add it, it still looks okay in pycharm and it definitely looks better in vscode, so well me what you think

Reply:
Either way is fine with me. How about we leave it as is and get Bar's opinion, then if we decide to switch to markdown-style links we can update all the links at once